app/src/App.tsx

```diff
@@ -1,62 +1,74 @@
 import i18n from "i18next";
-import { initReactI18next } from "react-i18next";
-import { createTheme, ThemeProvider } from "@mui/material/styles";
-import { red } from "@mui/material/colors";
-import { BrowserRouter, Navigate, Route, Routes } from "react-router-dom";
-import { Base } from "./components/Base";
-import { getLocaleOrFallback, resources, SAVED_LOCALE } from "./locale";
-import { Home } from "./pages/Home";
-import { About } from "./pages/About";
-import { Organization } from "./pages/Organization";
-import { FAQ } from "./pages/FAQ";
+import {initReactI18next} from "react-i18next";
```

Comment:
As mentioned in your other PR, please rebase and use the Prettier config that was just merged in. There shouldn't be formatting changes that are inconsistent with the existing files :)

Reply:
I'll continue reviewing the React code one this is complete.